activity-metrics/open-issues.md

```diff
@@ -1,10 +1,30 @@
-# Open Issues
-
 ## 1. Description
-Number of open issues
+
+These are the issues considered 'open' at some given point in time. The definition of 'open' differs from platform to platform and should be pre-defined.
+
+Note: do not confuse this with the opened tickets in a specific period of time regardless their current status. The open issues metric is a snapshot while the opened issues one needs of a period of time.
+
+### Parameters
+
+Mandatory: 
+* Point in time. This is the time for which the snapshot of the open issues is computed.

```

Comment:
"Point in Time" : Day of Analysis/Reporting ... Timestamp ... so, its "open issues as of September 9, 2018 at 2:30am CET.

Reply:
Yes, but not "time of reporting", because you could compute the metric for some point in the past.